compute/compute_test.go

```diff
@@ -0,0 +1,28 @@
+package snippets
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestComputeSnippets(t *testing.T) {
+	tc := testutil.SystemTest(t)
```

Comment:
This test assumes resources exist, which I don't think they will? Should there be a test main the sets up and tears down some resources?